### ISSUE TITLE
Automated Slider added in Home 

### ISSUE THREAD

**Issue by WebVishal** [Feminine-Divine/periodo#118]
Hii @khushishikhu  I think for a more attractive home page, we should add the automatic slider with animation text if it is good then Can I work on this idea as a Gssoc'21 participant

**Comment by khushishikhu** [comment]
@WebVishal yes you can . Thank you for opening this issue. Assigning this to you✌️

**Comment by anushri20** [comment]
@khushishikhu Hi, is this issue still open?

**Comment by khushishikhu** [comment]
@anushri20 No, You can look for any other issue. Where no one is working.

**Comment by satavshreya** [comment]
@khushishikhu @Aaishpra  is this issue open? I can work on this :)

**Comment by Aaishpra** [comment]
@anushri20 are u still interested in the issue or we can assign the next person

**Comment by satavshreya** [comment]
@Aaishpra @khushishikhu let me know abt this

**Comment by khushishikhu** [comment]
@satavshreya someone has already made a PR for this

**Comment by khushishikhu** [comment]
@satavshreya are you interested working on this

**Comment by satavshreya** [comment]
@khushishikhu yes ofcourse.The database  is taking time and Darkmode where do i write js code and they will be done .I am intereted for this one Do assign me

**Comment by satavshreya** [comment]
![image](https://user-images.githubusercontent.com/61575173/115847070-2fb22b80-a440-11eb-9a08-7d490f9f2906.png)
![image](https://user-images.githubusercontent.com/61575173/115847252-66884180-a440-11eb-940b-b81f797d0fa3.png)
![image](https://user-images.githubusercontent.com/61575173/115847301-730c9a00-a440-11eb-81c4-3bb76b2e1b08.png)
@khushishikhu @Aaishpra these pictures i have selected if any changes do tell me and alos i have kept the height of the slider like this if you want a change i am ready for that!

**Comment by khushishikhu** [comment]
@satavshreya can you write different quotes or statements in every slide

**Comment by satavshreya** [comment]
@khushishikhu yes chalega ! just confirm me about the slider height its fine right?

**Comment by khushishikhu** [comment]
@satavshreya can you show me how does the whole page looks like after scrolling down as well

**Comment by satavshreya** [comment]
![image](https://user-images.githubusercontent.com/61575173/115854094-88d18d80-a447-11eb-8b54-f5ed77132c84.png)
@khushishikhu

**Comment by khushishikhu** [comment]
@satavshreya great. Please move forward to make a PR

**Comment by subhangi2731** [comment]
@khushishikhu is this issue open ? can i work on this
